antora-playbook.yml

```diff
@@ -12,6 +12,7 @@ urls:
 ui:
   bundle:
     url: https://tiny-cloud-docs-antora-themes-production.s3.amazonaws.com/ui-bundle.zip
+    snapshot: true
```

Comment:
Leaving comment for reference: [remote-bundle#snapshot: true](https://docs.antora.org/antora/latest/playbook/ui-bundle-url/#remote-bundle), nice work finding this fix.